spec/models/reviewable_spec.rb

```diff
@@ -244,6 +244,55 @@
     end
   end
 
+  context ".recent_list_with_pending_first" do
+    fab!(:pending_reviewable1) { Fabricate(:reviewable, score: 150) }
+    fab!(:pending_reviewable2) { Fabricate(:reviewable, score: 100) }
+    fab!(:approved_reviewable1) { Fabricate(:reviewable, score: 300) }
+    fab!(:approved_reviewable2) { Fabricate(:reviewable, score: 200) }
+
+    fab!(:admin) { Fabricate(:admin) }
+
+    it "returns a list of reviewables with pending items first" do
+      list = Reviewable.recent_list_with_pending_first(admin)
+      expect(list).to contain_exactly(
+        pending_reviewable1,
+        pending_reviewable2,
+        approved_reviewable1,
+        approved_reviewable2,
+      )
+
+      pending_reviewable1.update!(status: Reviewable.statuses[:rejected])
```

Comment:
Minor but I wonder if we can just set this to rejected from the start. The variable is now confusing because this reviewable is no longer pending.

Reply:
I've introduced new variable `rejected_reviewable` that's used from the point the reviewable's status is updated to rejected. I think that should eliminate any confusion.

Reply:
:+1: